plugins/log/Cargo.toml

```diff
@@ -13,6 +13,13 @@ links = "tauri-plugin-log"
 rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.platforms.support]
+windows = { level = "full", notes = "" }
+linux = { level = "full", notes = "" }
+macos = { level = "full", notes = "" }
+android = { level = "unknown", notes = "" }
+ios = { level = "unknown", notes = "" }
```

Comment:
@lucasfernog can you also clarify on this one? It seems like log levels are not fully supported on mobile and I am not sure how well this is supported/works.